config.json

```diff
@@ -22,6 +22,7 @@
   "pushover_user" : null,
   "discord_webhook": null,
   "teams_webhook": null,
+  "operator_id": "CHANGEME",
```

Comment:
Set to null so we can do checks. People may not care for this in their responses

Reply:
Fixed up 🙂